db/migrate/20150910153517_add_full_name_to_custom_attributes.rb

```diff
@@ -0,0 +1,5 @@
+class AddFullNameToCustomAttributes < ActiveRecord::Migration
+  def change
+    add_column :custom_attributes, :full_name, :text
```

Comment:
This or use the already-existing `:source` column.  But, I don't know what `:source` was originally intended for.  @Fryguy?